controllers/app.controller.js

```diff
@@ -59,3 +62,24 @@ exports.getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postCommentByArticleId = (req, res, next) => {
+  const { username, body } = req.body;
+  const { article_id } = req.params;
+  const promises = [createCommentById(article_id, username, body)];
+  if (username) {
+    promises.push(usersData(username));
```

Comment:
There is no need to separately check if the user exists as a psql error will be thrown if it does not.